project/README.md.twig

```diff
@@ -15,6 +15,14 @@ Branch | Travis | Coveralls | Scrutinizer |
 {{ stable_branch }}   | [![Build Status][travis_stable_badge]][travis_stable_link]     | [![Coverage Status][coveralls_stable_badge]][coveralls_stable_link]     | [![Scrutinizer Status][scrutinizer_stable_badge]][scrutinizer_stable_link] |
 {{ unstable_branch }} | [![Build Status][travis_unstable_badge]][travis_unstable_link] | [![Coverage Status][coveralls_unstable_badge]][coveralls_unstable_link] | [![Scrutinizer Status][scrutinizer_unstable_badge]][scrutinizer_unstable_link] |
 
+{% if is_abandoned %}
+## WARNING: This bundle is abandoned
+
+Since maintainers have lost interest in this project, there is no active support on it.
```

Comment:
If all maintainers have lost interest in a project, but there still are a lot of contributions, would we drop support for it?

Reply:
I think it would be convenient to leave a call for those contributors to step up and become mantainers

Reply:
Right, I was trying to make a generic sentence. Do you have any suggestion ?
Should I only write `There is no active support`, then ?

Reply:
> I think it would be convenient to leave a call for those contributors to step up and become mantainers

That is what is done below, isn't it?

Reply:
Should I change the sentence then ? And how ?

I thought again, and I think it's ok.
- I ask for new maintainer in the next sentence.
- If we drop support on a project, we have indeed lost interest in it. I don't say that we'll abandon every project we lost interest in.

Reply:
You don't explicitely say it but it seems to be the implication.
```suggestion
There is no active support on it.
```

Reply:
Ok, I updated the message then.